socorro/lib/transform_rules.py

```diff
@@ -94,18 +147,22 @@ def action(self, *args, **kwargs):
         try:
             return self._action(*args, **kwargs)
         except KeyError, x:
-            self.config.logger.debug(
-                'Rule %s action failed because of missing key "%s"',
-                to_str(self.__class__),
-                x,
-            )
+            if not self._send_to_sentry('action', *args, **kwargs):
```

Comment:
If we're going to start treating `KeyError` as a "thing that should never happen", then we should squash this block and the next one into a big "catch all exceptions and log to sentry" block.